facdb/datasets.yml

```diff
@@ -7,3 +7,9 @@ datasets:
     version: latest
   - name: dca_operatingbusinesses
     version: 20210402
+  - name: dcla_culturalinstitutions
+    version: latest
```

Comment:
let's use explicit versioning

Reply:
https://data-library-app.herokuapp.com/?dataset=dcla_culturalinstitutions
and we can check them here